server/tracedb/azureappinsights.go

```diff
@@ -224,12 +326,8 @@ func parseAttributes(span *model.Span, row azquery.Row, index int) error {
 	return nil
 }
 
-func parseParentId(span *model.Span, row azquery.Row, index int) error {
-	if index == -1 {
-		return fmt.Errorf("parentId column not found")
-	}
-
-	rawParentId, ok := row[index].(string)
+func parseParentId(span *model.Span, value any) error {
+	rawParentId, ok := value.(string)
```

Comment:
```suggestion
	rawParentID, ok := value.(string)
```

Reply:
No. I exclude all `nil` values from the map before calling those functions